src/lib/work_partitioner/pending_zkapp_command.mli

```diff
@@ -0,0 +1,24 @@
+open Core_kernel
+open Snark_work_lib
+
+type t =
+  { job : (Spec.Single.t, Id.Single.t) With_job_meta.t
```

Comment:
The internal is exposed as we need to construct t. We could do otherwise and provide a constructor, wdyt?

Reply:
yes, in presense of mutable fields, I feel like constructor is a necessary addition